media/js/base/global.js

```diff
@@ -98,6 +98,11 @@ function isFirefox(userAgent) {
     );
 }
 
+function isSeaMonkey(userAgent) {
+    var ua = userAgent || navigator.userAgent;
+    return ((/SeaMonkey/i).test(ua));
```

Comment:
Nit: all the brackets might not be required here: could this just be `return /SeaMonkey/i.test(ua);`?